kaiax/gasless/config.go

```diff
@@ -26,38 +26,55 @@ var (
 		Name:     "gasless.allowed-tokens",
 		Usage:    "allow token addresses for gasless module, allow all tokens if all",
 		Value:    cli.NewStringSlice("all"),
-		Aliases:  []string{"genesis.module.gasless.allowed-tokens"},
+		Aliases:  []string{"kaiax.module.gasless.allowed-tokens"},
 		Category: "KAIAX",
 	}
 	DisableFlag = &cli.BoolFlag{
 		Name:     "gasless.disable",
 		Usage:    "disable gasless module",
 		Value:    false,
-		Aliases:  []string{"genesis.module.gasless.disable"},
+		Aliases:  []string{"kaiax.module.gasless.disable"},
+		Category: "KAIAX",
+	}
+	MaxGaslessBundleSizeFlag = &cli.IntFlag{
+		Name:     "gasless.max-gasless-bundle-size",
+		Usage:    "max size of gasless bundle, default value if 0, no limit if minus value",
+		Value:    100,
+		Aliases:  []string{"kaiax.module.gasless.max-gasless-bundle-size"},
 		Category: "KAIAX",
 	}
 )
 
 type GaslessConfig struct {
 	// all tokens are allowed if AllowedTokens is nil while all are disallowed if empty slice
-	AllowedTokens []common.Address `toml:",omitempty"`
-	Disable       bool
+	AllowedTokens        []common.Address `toml:",omitempty"`
+	Disable              bool
+	MaxGaslessBundleSize int
 }
 
-func GetGaslessConfig(ctx *cli.Context) *GaslessConfig {
-	allowedTokens := []common.Address(nil)
-	for _, addr := range ctx.StringSlice(AllowedTokensFlag.Name) {
-		if addr == "all" {
-			allowedTokens = nil
-			break
-		}
-		if allowedTokens == nil {
-			allowedTokens = []common.Address{}
+func DefaultGaslessConfig() *GaslessConfig {
+	return &GaslessConfig{
+		AllowedTokens:        nil,
+		Disable:              false,
+		MaxGaslessBundleSize: 100,
+	}
+}
+
+func SetGaslessConfig(ctx *cli.Context, cfg *GaslessConfig) {
+	if tokens := ctx.StringSlice(AllowedTokensFlag.Name); tokens != nil {
+		cfg.AllowedTokens = []common.Address{}
+		for _, addr := range tokens {
+			if addr == "all" {
+				cfg.AllowedTokens = nil
+				break
+			}
+			cfg.AllowedTokens = append(cfg.AllowedTokens, common.HexToAddress(addr))
 		}
-		allowedTokens = append(allowedTokens, common.HexToAddress(addr))
 	}
-	return &GaslessConfig{
-		AllowedTokens: allowedTokens,
-		Disable:       ctx.Bool(DisableFlag.Name),
+
+	cfg.Disable = ctx.Bool(DisableFlag.Name)
+
+	if size := ctx.Int(MaxGaslessBundleSizeFlag.Name); size != 0 {
+		cfg.MaxGaslessBundleSize = size
```

Comment:
if size < 0, set cfg.MaxGaslessBundleSize to maxint

Reply:
I'll change type into uint.